setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Setup script to help on *nix systems with local development prerequisites
+
+if ! command -v docker-compose &> /dev/null
```

Comment:
Need to deal with the fact the command is going to be 'docker compose' without a hyphen in some cases.  I know this because last night I dumped podman and reinstalled docker from the docker repository.

Reply:
Thanks for the tip! That lead me to [this question](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose) where it seems like `docker compose` (with a space) is the 'preferred' (by the Docker project) option. I'll look at how to update the script.

Reply:
good catch - this should be as simple as just `s/docker-compose/docker compose/` through the files

Reply:
It wasn't (it never is 😉). Changed the commands (including in scripts in `prod` folder) to use `docker compose` rather than `docker-compose` but didn't want to rename files (eg `docker-compose.yaml`).